test/data/ids.js

```diff
@@ -1,17 +1,28 @@
 const uuid = require('uuid/v4');
 
 module.exports = {
-  pil: {
-    grant: uuid(),
-    applied: uuid(),
-    rejected: uuid(),
-    transfer: uuid()
+  task: {
+    pil: {
+      grant: uuid(),
+      rejected: uuid(),
+      transfer: uuid()
+    },
+    place: {
+      applied: uuid(),
+      resolved: uuid()
+    },
+    project: {
+      grant: uuid()
+    }
   },
-  place: {
-    applied: uuid(),
-    resolved: uuid()
-  },
-  project: {
-    grant: uuid()
+  model: {
+    pil: {
+      applied: uuid(),
+      rejected: uuid()
+    },
+    place: {
+      applied: uuid(),
+      resolved: uuid()
+    }
```

Comment:
This was really confusing when I was writing the tests - some ids referred to task ids and some referred to model ids, so I've tried to put a bit more structure in there. They're still not great but at least it's less confusing now.